DataFormats/CTPPSDigi/src/classes_def.xml

```diff
@@ -14,7 +14,8 @@
   </class>
   <class name="edm::Wrapper<TotemTriggerCounters>"/>
 
-  <class name="TotemVFATStatus" ClassVersion="3">
+  <class name="TotemVFATStatus" ClassVersion="4">
+   <version ClassVersion="4" checksum="1562789855"/>
     <version ClassVersion="2" checksum="3003040825"/>
     <version ClassVersion="3" checksum="1439138260"/>
   </class>
```

Comment:
No particular reason for these to be dropped in the backport. Added them back manually to get in sync with `9_X_Y`.